src/pages/[post].tsx

```diff
@@ -261,60 +285,11 @@ export const getStaticProps: GetServerSideProps = async function ({params}) {
     }
   }
 
-  const {post, videoResource} = result
+  const {post, videoResource, tags} = result
   const lesson = await fetch(
     `${process.env.NEXT_PUBLIC_AUTH_DOMAIN}/api/v1/lessons/${post.fields.eggheadLessonId}`,
   ).then((res) => res.json())
 
-  const resource = {
-    id: `${post.fields.eggheadLessonId}`,
-    externalId: post.id,
-    title: post.fields.title,
-    slug: post.fields.slug,
-    summary: post.fields.description,
-    description: post.fields.body,
-    name: post.fields.title,
-    path: `/${post.fields.slug}`,
-    type: post.fields.postType,
-    ...(lesson && {
-      instructor_name: lesson.instructor?.full_name,
-      instructor: lesson.instructor,
-      image: lesson.image_480_url,
-    }),
-  }
-
-  let client = new Typesense.Client({
-    nodes: [
-      {
-        host: process.env.NEXT_PUBLIC_TYPESENSE_HOST!,
-        port: 443,
-        protocol: 'https',
-      },
-    ],
-    apiKey: process.env.TYPESENSE_WRITE_API_KEY!,
-    connectionTimeoutSeconds: 2,
-  })
-
-  if (
-    post.fields.state === 'published' &&
-    post.fields.visibility === 'public'
-  ) {
-    await client
-      .collections(process.env.TYPESENSE_COLLECTION_NAME!)
-      .documents()
-      .upsert({
-        ...resource,
-        published_at_timestamp: post.updatedAt.getTime(),
-      })
-      .catch((err) => {})
-  } else {
-    await client
-      .collections(process.env.TYPESENSE_COLLECTION_NAME!)
-      .documents()
-      .delete(resource.id)
-      .catch((err) => {})
-  }
```

Comment:
This Typesense indexing is out of sync with course builder and seems redundant. Currently it strip the indexed typesense objects of tags because when this was implemented we weren't indexing tags in posts..